iFixit/react-commerce
# Update product cross sell

close #1910 

## Note for CR

From #1910:
> Make sure the GA Events are copied over to the new cross-sell widget implementation

It seems to me that the tracking is entirely done with `useAddToCart('Frequently Bought Together')`. I haven't found any other custom tracking, but I'd appreciate a double check.

## QA

1. Open a [product page preview](https://react-commerce-git-update-product-cross-sell-ifixit.vercel.app/products/pro-tech-toolkit)
2. Verify that the frequently bought together box is rendered right after the add to cart button
3. Verify that you can successfully add "frequently bought together" items to cart
